apps/bfd-server-ng/src/main/java/gov/cms/bfd/server/ng/eob/EobHandler.java

```diff
@@ -61,8 +64,67 @@ public Bundle searchByBene(
     var eobs =
         claimRepository.findByBeneXrefSk(
             beneXrefSk.get(), serviceDate, lastUpdated, count, startIndex, sourceIds);
+
+    return getFhirEobs(eobs, true);
+  }
+
+  Bundle getFhirEobs(List<Claim> eobs, boolean filterSamhsa) {
+
+    List<Claim> filteredEobs = eobs;
```

Comment:
We try to use `var` to keep variable declarations simple.

```suggestion
    var filteredEobs = eobs;
```